app/views/users/show.html.erb

```diff
@@ -1 +1,26 @@
-<h1>This is the users:ID page</h1>
\ No newline at end of file
+<div class="user_posts">
+  <div class="user_profile">
+     <img src="<%= @user.photo %>" alt="user_pic" />
+     <div>
+        <p><%= @user.name %></p>
+        <p class="posts_count">Number of posts: <%= @user.posts_counter || 0 %></p>
+     </div>
+  </div>
+  <div class="user_bio">
+    <p><%= @user.bio || "no Bio provided by the user" %></p>
+  </div>
+  <ul>
+    <% @post.each do |post| %>
+      <% if @post.empty? %>
+        <p>User doesn't have posts yet</p>
+      <% else %>
+        <div>
+            <h2><%= post.title %></h2>
+            <p><%= post.text %></p>
+            <h3>Comment <%= post.comments.count%>,  <span>Likes <%= post.likes.count%></span></h3>
+        </div>
+      <% end %>
+    <% end %>
+  </ul>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/169436054-048bde5a-f590-42ca-8f17-3028f28a2f79.png)



- [ ] Kindly note that you need to follow the wireframes that were provided in the requirements, I know that the style does not matter but you need to add some basic CSS to follow the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md). check the wireframes :point_down: 

![image](https://user-images.githubusercontent.com/65068771/169435586-73bdb804-e322-44fe-a337-6f7ca8c310d0.png)